tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -44,7 +51,12 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |Main arg2
        |:: scala.collection.immutable
        |""".stripMargin,
-    topLines = Option(3)
+    topLines = Option(3),
+    compat = Map( // findPossibleDefaults and fillAllFields not yet supported
```

Comment:
This is not an issue in this test case I think so maybe remove the comment? Same above and below.